backend/controllers/referralControllers.ts

```diff
@@ -4,3 +4,23 @@ const prisma = new PrismaClient();
 export const createOneReferral = (dataClause: Prisma.ReferralCreateInput) => {
   return prisma.referral.create({ data: dataClause });
 };
+
+export const getReferralsFromUserId = (userId: string) =>{
```

Comment:
I think renaming this one to "getReferralsFromEmployeeId" would explain this function better.

Reply:
It took me a hot minute to find out if these are referrals i submitted or people submitted for me. Maybe `getSubmittedReferrals`?

Reply:
if these are the referrals people submitted for me (or input user), then we can use "getReferralsFromCandidateId" I think.

Reply:
Sure, but they're not